test/common/fit_common.py

```diff
@@ -212,24 +212,18 @@ def add_globals():
     global VERBOSITY
 
     # set api port and protocol from command line
-    if fitargs()['port'] != "None":
-        API_PORT = fitargs()['port']
-
-    if fitargs()['http'] == "True":
+    if fitargs()['http'] is True:
         API_PROTOCOL = "http"
-        if API_PORT == "None":
-            API_PORT = fitports()['http']
-
-    if fitargs()['https'] == "True":
+        API_PORT = str(fitports()['http'])
```

Comment:
So, the logic says that -http will supersede -https if both are specified. Is this intentional?  arg_parse has a mutually exclusive capability that will prevent this case from happening at all.